### PR TITLE
send feedback when a recognized command is found

### PR DESCRIPTION
**Pull request by mmiller1**
This is pretty basic, and while only tested with slack, it seems like it should be adapter agnostic. it provides all of the feedback I wanted/needed in #186

### CODE REVIEW

File: adapter/adapter.go [hunk 1]
```diff
@@ -511,6 +511,13 @@ func TriggerCommand(ctx context.Context, rawCommand string, id RequestorIdentity
 
 		return nil, fmt.Errorf("command lookup error: %w", err)
 	}
+	cmdFoundMessage := fmt.Sprintf("executing command: %s", cmdEntry.Command.Name)
```
Comment (clockworksoul) [inline adapter/adapter.go:514]: Would you mind capitalizing `executing`?

**Comment by clockworksoul** [comment]
Thanks for this!

Would you mind just quickly:
1. Capitalizing the word `executing`, and
2. Merging this into the `develop` branch instead of `main`?

**Comment by clockworksoul** [comment]
>Merging this into the `develop` branch instead of `main`?

If this makes for a more ugly merge, I'm also not entirely opposed to merging this into `main`. It's not a major change.

**Comment by mmiller1** [comment]
sure thing, I believe both comments have been addressed.

**Pull request merged**